server/apps/archive/archive.py

```diff
@@ -165,6 +167,12 @@ def on_create(self, docs):
             if not doc.get('ingest_provider'):
                 doc['source'] = DEFAULT_SOURCE_VALUE_FOR_MANUAL_ARTICLES
 
+            if 'urgency' not in doc:
+                doc['urgency'] = DEFAULT_URGENCY_VALUE_FOR_MANUAL_ARTICLES
+
+            if 'priority' not in doc:
+                doc['priority'] = DEFAULT_PRIORITY_VALUE_FOR_MANUAL_ARTICLES
```

Comment:
Tip: the same can be achieved in a more concise way by using the `dict`'s [setdefault()](https://docs.python.org/3.4/library/stdtypes.html#dict.setdefault) method. Example:
```py
doc.setdefault('priority', DEFAULT_PRIORITY_VALUE_FOR_MANUAL_ARTICLES)
```